catch/include/hip_test_defgroups.hh

```diff
@@ -95,8 +97,46 @@ THE SOFTWARE.
 
 /**
 * @defgroup KernelTest Kernel Functions Management
+* @{
+* This section describes the various kernel functions invocation.
+* @}
+*/
+
+/**
+ * @defgroup AtomicsTest Device Atomics
+ * @{
+ * This section describes tests for the Device Atomic APIs.
+ * @}
+ */
+
+ /**
+ * @addtogroup atomicExch atomicExch
  * @{
- * This section describes the various kernel functions invocation.
+ * @ingroup AtomicsTest
+ */
+
+/**
+ * Test Description
+ * ------------------------
+ *  - Compiles atomicExch with invalid parameters.
+ *  - Compiles the source with specialized Python tool.
+ *    -# Utilizes sub-process to invoke compilation of faulty source.
+ *    -# Performs post-processing of compiler output and counts errors.
+ * Test source
+ * ------------------------
+ *  - unit/atomics/CMakeLists.txt
+ * Test requirements
+ * ------------------------
+ *  - HIP_VERSION >= 5.2
+ */
+TEST_CASE("Unit_atomicExch_Negative_Parameters") {}
```

Comment:
@mirza-halilcevic, due to this line we are getting multiple definition error:
<img width="425" alt="multiple_definition_error" src="https://github.com/ROCm-Developer-Tools/hip-tests/assets/137397847/2e8cbcd3-e292-4090-bece-e6ad27f02604">

Will you please check?